File: app/src/main/java/me/blog/korn123/easydiary/views/LabelLayout.java
package me.blog.korn123.easydiary.views;

import android.content.Context;
import android.content.res.ColorStateList;
import android.support.annotation.Nullable;
import android.util.AttributeSet;
import android.widget.LinearLayout;

/**
 * Created by CHO HANJOONG on 2017-12-19.
 */

public class LabelLayout extends LinearLayout {
    public LabelLayout(Context context) {
        super(context);
    }

    public LabelLayout(Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
    }

    public LabelLayout(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    public void setColors(int textColor, int accentColor, int backgroundColor) {
        setBackgroundColor(accentColor);
    }
}

File: app/src/main/java/me/blog/korn123/easydiary/views/LabelLayout.kt
package me.blog.korn123.easydiary.views

import android.content.Context
import android.util.AttributeSet
import android.widget.LinearLayout

class LabelLayout : LinearLayout {
    constructor(context: Context) : super(context)

    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs)

    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr)

    fun setColors(textColor: Int, accentColor: Int, backgroundColor: Int) {
        setBackgroundColor(accentColor)
    }
}